todo_app/data/mongodb.py

```diff
@@ -7,7 +7,7 @@
 
 class MongoDB:
     def __init__(self):
-        self.client = pymongo.MongoClient(f"mongodb+srv://{os.getenv('MONGODB_USERNAME')}:{os.getenv('MONGODB_PASSWORD')}@cluster0.illz4.mongodb.net/{os.getenv('MONGODB_DATABASE_NAME')}?retryWrites=true&w=majority")
+        self.client = pymongo.MongoClient(os.getenv('MONGODB_CONNECTION_STRING'))
```

Comment:
I would expect this to change to also include some corresponding updates - `.env.test`, `.env.template`, the variables passed in for `.travis.yml` etc.